packages/devtools-launchpad/webpack.config.devtools.js

```diff
@@ -22,9 +22,7 @@ module.exports = (webpackConfig, envConfig) => {
   }
 
   webpackConfig.devtool = false;
-  webpackConfig.recordsPath = path.join(
-    rootDir, "assets/module-manifest.json"
-  );
+  webpackConfig.recordsPath = path.join(os.tmpdir(), "module-manifest.json");
```

Comment:
Ah, I wasn't really sure of its purpose. 😅 The main issue I was seeing is that this config from `devtools-launchpad` was always placing the file at the root of `devtools-core` (outside any packages), which seems wrong. 

The Webpack [docs][1] led me to think it was only used for hot module reloading, which is why I went down the temporary file route.

If we do in fact want it it to be checked in, we'd want the location to be based on the path to actual package in use (debugger, source map, etc.) not the launchpad. @juliandescottes, should I pass the current package path down to this launchpad config? Or override the path from the source map package? Something else...?

[1]: https://webpack.github.io/docs/configuration.html#recordspath-recordsinputpath-recordsoutputpath